src/utils/components/ListPageFilter/ListPageFilter.tsx

```diff
@@ -135,107 +172,96 @@ const ListPageFilter: FC<ListPageFilterProps> = ({
             selectedRowFilters={selectedRowFilters}
             updateRowFilterSelected={updateRowFilterSelected}
           />
-          <ToolbarItem className="co-filter-search--full-width">
-            <ToolbarFilter
-              deleteChip={(f, chip: string) => {
-                const newLabels = textFilters.labels.filter((label) => label !== chip);
-                applyTextFilters(STATIC_SEARCH_FILTERS.labels, newLabels.join(','));
-              }}
-              deleteChipGroup={() => {
-                applyTextFilters(STATIC_SEARCH_FILTERS.labels);
-              }}
-              categoryName={STATIC_SEARCH_FILTERS_LABELS.labels}
-              chips={textFilters.labels ?? []}
-            >
-              {searchFilters.map((filter) => (
+          {showSearchFilters && (
+            <ToolbarItem className="co-filter-search--full-width">
+              <ToolbarFilter
+                deleteChip={(f, chip: string) => {
+                  const newLabels = textFilters.labels.filter((label) => label !== chip);
+                  applyTextFilters(STATIC_SEARCH_FILTERS.labels, newLabels.join(','));
+                }}
+                deleteChipGroup={() => {
+                  applyTextFilters(STATIC_SEARCH_FILTERS.labels);
+                }}
+                categoryName={STATIC_SEARCH_FILTERS_LABELS.labels}
+                chips={textFilters.labels ?? []}
+              >
+                {searchFilters.map((filter) => (
+                  <ToolbarFilter
+                    deleteChip={() => {
+                      applyTextFilters(filter.type);
+                      searchType === filter.type && setSearchInputText('');
+                    }}
+                    categoryName={filter.filterGroupName}
+                    chips={textFilters[filter.type] ? [textFilters[filter.type]] : []}
+                    key={filter.type}
+                  >
+                    <></>
```

Comment:
need that?

Reply:
yes. `ToolbarFilter` requires a children